manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -937,6 +937,7 @@ metadata:
   namespace: istio-system
   labels:
     istio.io/rev: default
+    istio.io/default: "true"
```

Comment:
How about `istio.io/rev.default` or `istio.io/rev-default`? It took me a while to associate this `istio.io/default` with the revision configured earlier.

Reply:
I am still not sure why we need this. Previously it was mentioned to distnguish default vs revision items - but we already know this by their name

Reply:
Yeah my thinking was that it'd be nice to have a consistent way of getting objects associated with `defaultRevision` across all resource types rather than relying on name but this may be solving for a problem we don't actually have.

Reply:
Yeah I see the idea, and it makes sense, but there is also additional cost/risk of confusion/complexity of introducing a new label. In this case, the cost and the benefit is low, so I don't have a strong opinion either way.